src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -150,6 +161,18 @@ uint16_t CaptureOptionsDialog::GetMaxCopyRawStackSize() const {
   return result;
 }
 
+void CaptureOptionsDialog::SetThreadStateChangeCallstackMaxCopyRawStackSize(
+    uint16_t stack_dump_size) {
+  ORBIT_CHECK(stack_dump_size % 8 == 0);
```

Comment:
ORBIT_CHECK? Are we going to crash if the user sets a wrong stack_size?

Shouldn't instead 
1- Log an error with `ORBIT_ERROR()`
2- Take the next multiple of 8?